opal/util/info.c

```diff
@@ -67,12 +67,14 @@ OBJ_CLASS_INSTANCE(opal_info_entry_t, opal_list_item_t, info_entry_constructor,
 /*
  * Duplicate an info
```

Comment:
This is a major change, it deserves better documentation.

Reply:
Fixed